src/content/stakeholders/account-holders.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Card/Account Holder Responsibilities
-description: "Account holders must comply with agency regulations and rules, maintain records, and keep their account secure."
-intro: "The account holder, also known as the cardholder, is the individual or agency/organization component designated by an agency/organization to receive a GSA SmartPay® account."
+description: Account holders must comply with agency regulations and rules, maintain records, and keep their account secure.
```

Comment:
The program wants to use `card/account holder` in most instances, but I think here we might want to use `card and account holders` or simply `account holders` because I'm not convinced that search bots are going to be able to parse `card/account` as effectively as a human might.

Reply:
I agree, I just wasn't sure if there was some precedent. Fine to change it to: Discover card and account holder responsibilities; how they can comply with agency regulations and rules, maintain records, and keep their account secure.